Algorithms/collatz-conj.py

```diff
@@ -0,0 +1,11 @@
+num = int(input("Enter a number: ")) #prompt user to enter a number
+def collatz_con(n):
```

Comment:
let's talk about naming functions 💅

Reply:
Listening

Reply:
At least let the name define what the function does and it's always good practice to avoid the shortening of names.